client.go

```diff
@@ -328,7 +338,7 @@ func (r *marathonClient) buildAPIRequest(method, uri string, reader io.Reader) (
 	// Build the HTTP request to Marathon
 	request, err = r.client.buildMarathonRequest(method, member, uri, reader)
 	if err != nil {
-		return nil, member, err
+		return nil, member, newRequestError{err}
```

Comment:
Not for reviewers: `buildAPIRequest` is only used on 2 places. In `apiCall` and `connectToSSE`. In order to be sure that we don't break anything with changing the error type here, I "unwrap" the error in the `apiCall` once again to keep the exact same behaviour as before. Those errors seem to be propagated until to the user. I am not 100% sure whether we really have to do this though.